NDTensors/src/dense/tensoralgebra/contract.jl

```diff
@@ -345,7 +345,8 @@ function _contract!(
   tA = 'N'
   if props.permuteA
     #@timeit_debug timer "_contract!: permutedims A" begin
-    @strided Ap = permutedims(AT, props.PA)
+    #@strided 
+    Ap = permutedims(AT, props.PA)
```

Comment:
Reminder to make `NDTensors.permutedims` that dispatches on `Array` or `CuArray` to call `@strided` on `Array`.